### ISSUE TITLE
AttributeError: 'COSConfigCharm' object has no attribute '_tracing'

### ISSUE THREAD

**Issue by gatici** [canonical/cos-configuration-k8s-operator#114]
### Bug Description

After deploying Cos-lite stack, cos-configuration blocked with the following error.

```
Model                  Controller                         Cloud/Region                                   Version  SLA          Timestamp
prod-telco-5g-cos-tel  juju-controller-35-production-tel  k8s-prod-telco-5g-core-tel-microk8s/localhost  3.5.4    unsupported  12:48:43Z

App                Version  Status   Scale  Charm                  Channel        Rev  Address         Exposed  Message
alertmanager       0.27.0   active       1  alertmanager-k8s       latest/stable  125  10.152.183.194  no       
catalogue                   active       1  catalogue-k8s          latest/stable   59  10.152.183.107  no       
cos-configuration  3.5.0    blocked      1  cos-configuration-k8s  latest/stable   58  10.152.183.237  no       Sync failed: Sync error: Exited with code 1.
grafana            9.5.3    active       1  grafana-k8s            latest/stable  117  10.152.183.32   no       
loki               2.9.6    active       1  loki-k8s               latest/stable  160  10.152.183.236  no       
prometheus         2.52.0   active       1  prometheus-k8s         latest/stable  210  10.152.183.36   no       
traefik            2.11.0   active       1  traefik-k8s            latest/stable  199  10.152.183.204  no       Serving at 10.15.2.56

Unit                  Workload  Agent  Address       Ports  Message
alertmanager/0*       active    idle   10.1.172.89          
catalogue/0*          active    idle   10.1.108.149         
cos-configuration/0*  blocked   idle   10.1.18.87           Sync failed: Sync error: Exited with code 1.
grafana/0*            active    idle   10.1.202.154         
loki/0*               active    idle   10.1.73.151          
prometheus/0*         active    idle   10.1.108.216         
traefik/0*            active    idle   10.1.18.89           Serving at 10.15.2.56

Offer       Application  Charm           Rev  Connected  Endpoint              Interface                Role
loki        loki         loki-k8s        160  0/0        logging               loki_push_api            provider
prometheus  prometheus   prometheus-k8s  210  0/0        receive-remote-write  prometheus_remote_write  provider

```

### To Reproduce

Deploy the following main.tf file:

data "juju_model" "cos" {
  name =  model_name
}

module "cos" {
  source                   = "git::https://github.com/canonical/terraform-juju-sdcore//modules/external/cos-lite"
  model_name               = data.juju_model.cos.name
  deploy_cos_configuration = true
  cos_configuration_config = {
    git_repo                = "https://github.com/canonical/sdcore-cos-configuration"
    git_branch              = "main"
    grafana_dashboards_path = "grafana_dashboards/sdcore/"
  }
}

### Environment

Juju 3.5
Microk8s 1.31

### Relevant log output

```shell
unit-cos-configuration-0: 08:36:14 INFO juju.worker.uniter.operation ran "content-from-git-storage-detaching" hook (via hook dispatching script: dispatch)
unit-cos-configuration-0: 08:36:14 ERROR unit.cos-configuration/0.juju-log prometheus-config:13: Uncaught exception while in charm code:
Traceback (most recent call last):
  File "./src/charm.py", line 489, in <module>
    main(COSConfigCharm, use_juju_for_storage=True)
  File "/var/lib/juju/agents/unit-cos-configuration-0/charm/venv/ops/main.py", line 549, in main
    manager = _Manager(charm_class, use_juju_for_storage=use_juju_for_storage)
  File "/var/lib/juju/agents/unit-cos-configuration-0/charm/venv/ops/main.py", line 432, in __init__
    self.charm = self._make_charm(self.framework, self.dispatcher)
  File "/var/lib/juju/agents/unit-cos-configuration-0/charm/venv/ops/main.py", line 435, in _make_charm
    charm = self._charm_class(framework)
  File "/var/lib/juju/agents/unit-cos-configuration-0/charm/lib/charms/tempo_k8s/v1/charm_tracing.py", line 439, in wrap_init
    tracing_endpoint = _get_tracing_endpoint(tracing_endpoint_attr, self, charm_type)
  File "/var/lib/juju/agents/unit-cos-configuration-0/charm/lib/charms/tempo_k8s/v1/charm_tracing.py", line 321, in _get_tracing_endpoint
    _tracing_endpoint = getattr(charm_instance, tracing_endpoint_attr)
  File "./src/charm.py", line 483, in tracing_endpoint
    if self._tracing.is_ready():
AttributeError: 'COSConfigCharm' object has no attribute '_tracing'
unit-cos-configuration-0: 08:36:14 ERROR juju.worker.uniter.operation hook "prometheus-config-relation-departed" (via hook dispatching script: dispatch) failed: exit status 1
unit-cos-configuration-0: 08:36:14 INFO juju.worker.uniter awaiting error resolution for "relation-departed" hook
```

**Comment by gaborbk** [comment]
I would like to add my +1 to this issue, I have also been seeing it in my deployment of `cos-lite` with `cos-configuration-k8s`

**Comment by mmkay** [comment]
Should already be fixed by #109. If you still experience it with revision 66 please feel free to reopen.
